src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/OpenSslCachedSystemStoreProvider.cs

```diff
@@ -307,7 +313,6 @@ bool ProcessFile(string file, out DateTime lastModified, bool skipStat = false)
             // In order to maintain "finalization-free" the GetNativeCollections method would need to
             // DangerousAddRef, and the callers would need to DangerousRelease, adding more interlocked operations
             // on every call.
-
```

Comment:
The blank line shouldn't have been removed, since the comment isn't about the following code, it's about code that's not present; making it a separate logical code paragraph.  But it's not worth spinning on.